float32 l_angle #NEED TO DEFINE INTREVALS
float32 r_angle
string emotion
